doc/reference/views.rst

```diff
@@ -1072,6 +1072,8 @@ attributes:
 ``stacked``
   only used for ``bar`` charts. If present and set to ``True``, stacks bars
   within a group
+``disable_linking``
```

Comment:
Need to add this attribute to `graph_view.rng`

Reply:
bien vu...